lib/simulation/openfisca/mapping/ressources.js

```diff
@@ -54,5 +54,8 @@ exports.famille = {
 };
 
 exports.foyerFiscal = {
-    pensions_alimentaires_versees: 'rncPensionsAlimentairesVersees'
+    pensions_alimentaires_versees: {
+        src: 'rncPensionsAlimentairesVersees',
+        fn: function(value) { return -value; }
+        }
```

Comment:
Nope, that “hanging indent” is used by the OpenFisca team in Python, but in JS we unindent the closing brackets & parens.